src/api/mod.rs

```diff
@@ -117,6 +117,8 @@ impl From<Hash> for String {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Stats {
   pub tick: u64,
+  pub mana: u64,
+  pub size: i128
```

Comment:
We can't have `u128`s or `i128`s on Serialize-derived struct as `serde_json` does not support them and annoyingly crashes.